invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/License/LicenseModal.js

```diff
@@ -222,21 +221,29 @@ export class LicenseModal extends Component {
                 <Form>
                   <TextField
                     label={i18next.t("Title")}
-                    placeholder={i18next.t("License title")}
                     fieldPath="selectedLicense.title"
                     required
                     // eslint-disable-next-line
                     autoFocus
                   />
-                  <TextAreaField
-                    fieldPath="selectedLicense.description"
+                  <Overridable
+                    id="InvenioRdmRecords.DepositForm.LicenseField.Modal.description"
```

Comment:
We can remove the overridable ids for these internal ones as we added one for the top level field.